pyaerocom/plot/mapping.py

```diff
@@ -556,7 +562,7 @@ def plot_nmb_map_colocateddata(coldata, in_percent=True, vmin=-100,
         valid = ~stacked.isnull()
         coords = stacked.latlon[valid].values
         lats, lons = list(zip(*list(coords)))
-        data = stacked.data[valid]
+        data = stacked.data[tuple(valid)]
```

Comment:
Why `tuple(valid)`?

Reply:
`valid` can not be a list, it needs to be a tuple in order to avoid a numpy FutureWarning

Reply:
this was the warning message, shown on CI

```
tests/plot/test_mapping.py::test_plot_nmb_map_colocateddata4D
  /home/runner/work/pyaerocom/pyaerocom/pyaerocom/plot/mapping.py:559: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
```

Reply:
ok